plugin/dns64/dns64.go

```diff
@@ -36,7 +36,11 @@ type DNS64 struct {
 func (d *DNS64) ServeDNS(ctx context.Context, w dns.ResponseWriter, r *dns.Msg) (int, error) {
 	// Don't proxy if we don't need to.
 	if !d.requestShouldIntercept(&request.Request{W: w, Req: r}) {
-		return d.Next.ServeDNS(ctx, w, r)
```

Comment:
I think we should use plugin.NextOrFailure() here. 

https://github.com/coredns/coredns/blob/e3f83cb1fabb9b1cbaffb9df3c4b65476e92c39b/plugin/plugin.go#L73

Reply:
Makes sense, applied!